python/cog/server/worker.py

```diff
@@ -221,13 +225,14 @@ def _predict(self, payload: Dict[str, Any]) -> None:
             result = predict(**payload)
 
             if result:
-                if isinstance(result, types.GeneratorType):
-                    self._events.send(PredictionOutputType(multi=True))
-                    for r in result:
-                        self._events.send(PredictionOutput(payload=make_encodeable(r)))
-                else:
-                    self._events.send(PredictionOutputType(multi=False))
-                    self._events.send(PredictionOutput(payload=make_encodeable(result)))
+                with self._events_lock:
+                    if isinstance(result, types.GeneratorType):
+                        self._events.send(PredictionOutputType(multi=True))
+                        for r in result:
+                            self._events.send(PredictionOutput(payload=make_encodeable(r)))
+                    else:
+                        self._events.send(PredictionOutputType(multi=False))
+                        self._events.send(PredictionOutput(payload=make_encodeable(result)))
```

Comment:
Fix formatting error (`python -m ruff format python`)

```suggestion
                with self._events_lock:
                    if isinstance(result, types.GeneratorType):
                        self._events.send(PredictionOutputType(multi=True))
                        for r in result:
                            self._events.send(
                                PredictionOutput(payload=make_encodeable(r))
                            )
                    else:
                        self._events.send(PredictionOutputType(multi=False))
                        self._events.send(
                            PredictionOutput(payload=make_encodeable(result))
                        )
```

Reply:
Thanks, I have reformatted the code.